### ISSUE TITLE
Virus nel file di aggiornamento 8.4.7 – 06/07/2023

### ISSUE THREAD

**Issue by FabioB85** [SHWotever/SimHub#1362]
![Nuovo Immagine bitmap](https://github.com/SHWotever/SimHub/assets/138779703/e8564637-217a-45eb-82b6-5963993c9edc)

Rilevato virus nel file di aggiornamento GRAVE!

**Comment by SHWotever** [comment]
Hi ! This is a false positive, defender recently is having much more of them :  https://github.com/SHWotever/SimHub/wiki/SimHub-Troubleshootings#simhubexe-disappears-or-antivirus-detects-simhub

I submited it to microsoft to make them fix the wrong detection, it sohuld take a few hours.
As you can see virus total shows it as perfectly fine over 48 antiviruses : https://www.virustotal.com/gui/file/a67510a18cfc4fbef44d52688b9c019145a1c28796f874bea82c0b3f7c097613

**Comment by FabioB85** [comment]
Are you offended if... I prefer the antivirus to see it clean, I'll wait for tomorrow... I hope you understand!?


Inviato da Posta<https://go.microsoft.com/fwlink/?LinkId=550986> per Windows

Da: ***@***.***>
Inviato: giovedì 6 luglio 2023 13:29
A: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Oggetto: Re: [SHWotever/SimHub] Virus nel file di aggiornamento 8.4.7 – 06/07/2023 (Issue #1362)


Hi ! This is a false positive, defender recently is having much more of them : https://github.com/SHWotever/SimHub/wiki/SimHub-Troubleshootings#simhubexe-disappears-or-antivirus-detects-simhub

I submited it to microsoft to make them fix the wrong detection, it sohuld take a few hours.
As you can see virus total shows it as perfectly fine over 48 antiviruses : https://www.virustotal.com/gui/file/a67510a18cfc4fbef44d52688b9c019145a1c28796f874bea82c0b3f7c097613

**Comment by SHWotever** [comment]
Yeah I totally understand, False positives are a plague and I get that after such notification you can't be 100% sure,

Before trying again tomorow make sure to follow the microsoft instructions about updating and clearing the "cloud detection" caches (see Trigger an update) : 
https://www.microsoft.com/en-us/wdsi/defenderupdates

**Comment by FabioB85** [comment]
cd %ProgramFiles%\Windows Defender

MpCmdRun.exe -removedefinitions -dynamicsignatures

MpCmdRun.exe -SignatureUpdate
Thanks, I didn't know about this!

Inviato da Posta<https://go.microsoft.com/fwlink/?LinkId=550986> per Windows

Da: ***@***.***>
Inviato: giovedì 6 luglio 2023 13:47
A: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Oggetto: Re: [SHWotever/SimHub] Virus nel file di aggiornamento 8.4.7 – 06/07/2023 (Issue #1362)


Yeah I totally understand, False positives are a plague and I get that after such notification you can't be 100% sure,

Before trying again tomorow make sure to follow the microsoft instructions about updating and clearing the "cloud detection" caches (see Trigger an update) :
https://www.microsoft.com/en-us/wdsi/defenderupdates

**Comment by SHWotever** [comment]
Hi ! apparently the detection got cleared before MS analyzed the file : 
![image](https://github.com/SHWotever/SimHub/assets/2207331/a44ff03f-34c9-4138-917f-1da5687ea7f3)

you just need to refresh defender, as you pointed, running as administrator :
```bat
cd %ProgramFiles%\Windows Defender
MpCmdRun.exe -removedefinitions -dynamicsignatures
MpCmdRun.exe -SignatureUpdate
```


False positives mysteries ...

**Comment by FabioB85** [comment]
oK!...
Thank you Mister Boss!

Inviato da Posta<https://go.microsoft.com/fwlink/?LinkId=550986> per Windows

Da: ***@***.***>
Inviato: giovedì 6 luglio 2023 14:25
A: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Oggetto: Re: [SHWotever/SimHub] Virus nel file di aggiornamento 8.4.7 – 06/07/2023 (Issue #1362)


Closed #1362<https://github.com/SHWotever/SimHub/issues/1362> as completed.
